plugins/processors/starlark/starlark_test.go

```diff
@@ -3306,7 +3285,7 @@ func TestAllScriptTestData(t *testing.T) {
 				lines := strings.Split(string(b), "\n")
 				inputMetrics := parseMetricsFrom(t, lines, "Example Input:")
 				expectedErrorStr := parseErrorMessage(t, lines, "Example Output Error:")
-				outputMetrics := []telegraf.Metric{}
+				outputMetrics := make([]telegraf.Metric, 0)
```

Comment:
It doesn't make sense to initialize the array here as it will be overwritten anyway, does it?
```suggestion
				var outputMetrics []telegraf.Metric
```

Reply:
Changed.